superduperdb/container/document.py

```diff
@@ -124,14 +124,13 @@ def _decode(r: t.Dict, encoders: t.Dict) -> t.Any:
 def _encode(r: t.Any) -> t.Any:
     if isinstance(r, dict):
         return {k: _encode(v) for k, v in r.items()}
-    if isinstance(r, Encodable):
+    elif isinstance(r, Encodable):
```

Comment:
I don't think this will fix #968 (I think the logic is still the same here). See #970 for the likely fix.